PassleSync.Core/ContentFinders/PasslePostContentFinder.cs

```diff
@@ -9,8 +9,9 @@ public PasslePostContentFinder(ConfigService configService) : base(configService
         {
         }
 
-        protected override string UrlPrefix => _configService.PostPermalinkPrefix;
+        protected override string UrlTemplate => _configService.PostPermalinkTemplate;
         protected override string ShortcodeName => "PostShortcode";
+        protected override string SlugName => "PostSlug";
```

Comment:
I don't think this is necessary as per previous comments as we don't need to capture this